app/src/battery.c

```diff
@@ -85,7 +287,7 @@ static int zmk_battery_init(const struct device *_arg) {
     int rc = zmk_battery_update(battery);
 
     if (rc != 0) {
-        LOG_DBG("Failed to update battery value: %d.", rc);
+        LOG_WRN("Failed to update %d battery value(s) which the configured sensor supports.", rc);
```

Comment:
I'm not sure knowing how many channels failed is useful information. It would be much easier to know went wrong if you just logged which channels failed right after attempting to read them.

Reply:
you are right, its not useful... except the code that calls this function is expecting a 0 for success... so if that number is non-zero, the calling code handles the error. It seemed like a simple solution